tests/test_subclass_hierarchy.py

```diff
@@ -0,0 +1,40 @@
+import sys
+from rdflib import RDFS, Graph
+
+sys.path.append("..")
+from bricksrc.namespaces import BRICK
+
+g = Graph()
+g.parse("Brick.ttl", format="turtle")
+
+g.bind("rdfs", RDFS)
+g.bind("brick", BRICK)
+
+
+def minify(node):
+    return node.split("#")[-1]
+
+
+def test_cycles():
+    # Find all the classes that make the hierarchy cyclic.
+    classes_in_cycles = []
+    res1 = g.query(
+        "SELECT ?brick_class WHERE { ?brick_class rdfs:subClassOf+ ?brick_class .}"
+    )
+    for row in res1:
+        classes_in_cycles.append(row[0])
+
+    # Find all the triples involved.
+    loops = set()
+    for brick_class in classes_in_cycles:
+        res2 = g.query(
+            """SELECT DISTINCT ?s ?o WHERE {
```

Comment:
Stylistically, can you fold this string back to the line above?

```python
res2 = g.query("""SELECT # and so on
```

Reply:
`Black` formats it like that, I guess. Similar to queries on other files: https://github.com/BrickSchema/Brick/blob/f8c9a4d4ae2cb0206e8719b97b29245bab0239ec/tests/test_no_inference.py#L112

Reply:
Ok! Far be it from me to question Black's output :)